src/App.css

```diff
@@ -1,3 +1,20 @@
 .iiif-player > .container {
-  padding-top: 100px;
+  padding-top: 5px;
+}
+
+.iiif-player-demo > .row {
```

Comment:
Yes, that sounds good. For it to work in a CSS file from within demo directory, I think we might need a webpack.config in demo. I'm not sure about it. I will try it and see how it can be done.